app/helpers/application_helper.rb

```diff
@@ -93,17 +93,22 @@ def icon_url(name)
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
```

Comment:
The `_quality` and `_format` parameters are never used in this method.  Any thoughts on removing them?  I guess I could try it and see how many other tests break...